src/agent/onefuzz-task/src/tasks/fuzz/libfuzzer/common.rs

```diff
@@ -61,14 +67,23 @@ pub struct Config {
 
     #[serde(flatten)]
     pub common: CommonConfig,
+
+    #[serde(flatten)]
+    pub extra: L::Config,
```

Comment:
In the `libfuzzer-dotnet` task type, this will be a struct along the lines of:
```rust
struct LibFuzzerDotnetConfig {
    target_assembly: PathBuf,
    target_class: String,
    target_method: String,
}
```